src/m365/spo/commands/page/page-section-add.ts

```diff
@@ -19,11 +20,25 @@ interface Options extends GlobalOptions {
   order?: number;
   zoneEmphasis?: string;
   isLayoutReflowOnTop?: boolean;
+  isCollapsibleSection?: boolean;
+  showDivider?: boolean;
+  iconAlignment?: string;
+  isExpanded?: boolean;
+  gradientText?: string;
+  imageUrl?: string;
+  imageHeight?: number;
+  imageWidth?: number;
+  fillMode?: string;
+  useLightText?: boolean;
+  overlayColor?: string;
+  overlayOpacity?: number;
```

Comment:
Since we've a bunch of new options, let's try to type them as well in `#initTypes`

Reply:
Done